### PR TITLE
optimize talk Dockerfile

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: Containers/talk/Dockerfile [hunk 1]
```diff
@@ -30,15 +33,11 @@ RUN set -ex; \
         util-linux \
         build-base \
         lua5.3-dev \
-        luarocks5.3;
-
+        luarocks5.3; \
+    \
 # Give root a random password
```
Comment (szaimen) [inline Containers/talk/Dockerfile:38]: Is that possible?
```suggestion
    # Give root a random password
```

Comment (Zoey2936) [reply Containers/talk/Dockerfile:38]: yes

File: Containers/talk/Dockerfile [hunk 2]
```diff
@@ -30,15 +33,11 @@ RUN set -ex; \
...
-# Give root a random password
-RUN echo "root:$(openssl rand -base64 12)" | chpasswd
-
-COPY --chmod=775 start.sh /usr/bin/start.sh
-COPY --chmod=664 supervisord.conf /supervisord.conf
-
-RUN set -ex; \
+        luarocks5.3; \
+    \
+    # Give root a random password
```
Comment (szaimen) [inline Containers/talk/Dockerfile:38]:
```suggestion
# Give root a random password
```

**Review by szaimen — APPROVED** [review]
LGTM

**Comment by Zoey2936** [comment]
grafik
was this only this one PR?

**Comment by szaimen** [comment]
The talk build initially failed and I had to restart the failing job...

**Comment by szaimen** [comment]
Ah yes I suppose so... Crazy!

**Pull request merged**